nextflow.config

```diff
@@ -49,9 +49,11 @@ params {
   // Variant Calling
   ascat_ploidy = null // Use default value
   ascat_purity = null // Use default value
-  cf_coeff = "0.05"  // default value for Control-FREEC
-  cf_ploidy = "2"     // you can use 2,3,4
-  cf_window = null      // by default we are not using this in Control-FREEC
+  cf_coeff = 0.05                    // default value for Control-FREEC
+  cf_contamination = null            // by default not specified in Control-FREEC
+  cf_contamination_adjustment = null // by default we are not using this in Control-FREEC
+  cf_ploidy = "2"                    // you can use 2,3,4
+  cf_window = null                   // by default we are not using this in Control-FREEC
   generate_gvcf = null // g.vcf are not produced by HaplotypeCaller by default
```

Comment:
Seems like these parameters are added here but not applied anywhere so far. Is this intentional?

Reply:
Yes, totally, so that I don't forget about all of them.